tests/test_parametric_shapes/test_RotateStraightShape.py

```diff
@@ -222,6 +225,31 @@ def test_hash_value(self):
         assert self.test_shape.solid is not None
         assert initial_hash_value != self.test_shape.hash_value
 
+    def test_export_stp(self):
```

Comment:
There is another test_export_stp in this file.

Reply:
renamed test_export_stp method, thanks for spotting that